CRM/Contribute/Form/AbstractEditPayment.php

```diff
@@ -404,17 +404,6 @@ public function assignProcessors() {
     $this->assign_by_ref('paymentProcessor', $this->_paymentProcessor);
   }
 
-  /**
-   * Get current currency from DB or use default currency.
-   *
-   * @param array $submittedValues
-   *
-   * @return string
-   */
-  public function getCurrency($submittedValues = []) {
-    return $submittedValues['currency'] ?? $this->_values['currency'] ?? CRM_Core_Config::singleton()->defaultCurrency;
```

Comment:
the reference to `_values` appears to be copy & paste & not used by inheriting forms